src/openassets.es6

```diff
@@ -138,17 +140,17 @@ module.exports = function(robot) {
     return parseInt(assetDetails.balance) + parseInt(assetDetails.unconfirmed_balance);
   }
 
-  robot.hear(/kredits show (.+)/i, function(hearResponse) {
+  robot.hear(new RegExp(`${robotKeyword} show (.+)`, 'i'), function(hearResponse) {
     balanceOf(hearResponse.match[1], function(assetDetails) {
       if (assetDetails) {
-        hearResponse.send( hearResponse.match[1] + ' has ' + totalBalanceOfAsset(assetDetails) + ' kredits');
+        hearResponse.send( hearResponse.match[1] + ' has ' + totalBalanceOfAsset(assetDetails) + ' ' + robotKeyword);
```

Comment:
I think this should be "OA_ASSET_NAME" instead. Maybe we should even rename the bot keyword to asset name?